src/metatrain/utils/testing.py

```diff
@@ -0,0 +1,76 @@
+# This file contains some example TensorMap layouts that can be
+# used for testing purposes.
+
+import torch
+from metatensor.torch import Labels, TensorBlock, TensorMap
+
+
+block = TensorBlock(
+    # float64: otherwise metatensor can't serialize
+    values=torch.empty(0, 1, dtype=torch.float64),
+    samples=Labels(
+        names=["system"],
+        values=torch.empty((0, 1), dtype=torch.int32),
+    ),
+    components=[],
+    properties=Labels.range("energy", 1),
+)
+energy_layout = TensorMap(
```

Comment:
Please add documentation to these. You can add a doctring for variable with 

```py
blabla = Something(
    ...
)
"""Some doc string here"""
```

And then render them with sphinx.

Reply:
I tried but I get the following:
```
 File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/filippo/code/metatrain/.tox/docs/lib/python3.12/site-packages/metatrain/utils/testing.py", line 40, in <module>
    block_with_position_gradients.add_gradient("positions", position_gradient_block)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'add_gradient'
```
i.e. these seem to be "documentation" objects. SInce the testing module will already be gone in #376, can we do without the docs?

Reply:
ok, fair enough for now